Makefile

```
@@ -126,6 +126,9 @@ generate-doc: ## Generate the project's documentation
 generate-release-summary: ## Generate a release summary from Github's issues and PRs
 	@python scripts/generate_release_summary.py
 
+publication-watchdog-start: ## Start the publication watchdog
```

```suggestion
start-publication-watchdog: ## Start the publication watchdog
```

Je trouve que c'est mieux comme ça, c'est plus proche d'une phrase.

---

Ha ! J'avais fait ça au début, et puis je me suis dit que c'était mieux de respecter le pattern des autres commandes comme `zmd-start`.

Je viens de push le commit corrigé.